docs/Example HOWTO.md

```diff
@@ -120,24 +120,17 @@ _nmos-query._tcp        PTR     qry-api-1._nmos-query._tcp
 Now we add `SRV` records that return the URL for the registration and query servers.  In this case, both of the records point to `rds1.example.com`.
 
 ```
-; NMOS RDS services                          TTL     Class  SRV  Priority  Weight  Port  Target
-reg-api-1._nmos-register._tcp.example.com.     3600    IN     SRV  10        10      80    rds1.example.com.
-qry-api-1._nmos-query._tcp.example.com.        3600    IN     SRV  10        10      80    rds1.example.com.
+; NMOS RDS and Query Services       Type   Priority  Weight   Port   Host 
+reg-api-1._nmos-register._tcp        SRV     10        10      80    rds1.example.com.
+qry-api-1._nmos-query._tcp           SRV     10        10      80    rds1.example.com.
 ```
 
 We add `TXT` records which provide information relevant to the IS-04 specification
 
 ```
 ; Additional metadata relevant to the IS-04 specification. See IS-04 specification section "Discovery: Registered Operation"
-reg-api-1._nmos-register._tcp.example.com.        TXT     "api_ver=v1.0,v1.1,v1.2,v1.3" "api_proto=http" "pri=0" "api_auth=false"
-qry-api-1._nmos-query._tcp.example.com.           TXT     "api_ver=v1.0,v1.1,v1.2,v1.3" "api_proto=http" "pri=0" "api_auth=false"
```

Comment:
I see that you've removed the `.example.com.` suffix from many of the records. I'm not aware of when that's allowed/appropriate, please would you explain?

Reply:
I did a bit of experimenting and looks like we could remove all example.com.(s) from the database file and still dig works. Didn't go much further but for the interest of consistency I'll add in the example.com. for all the items in the db file. I'm thinking since the domain is defined in named.conf that BIND9 uses this for queries of the form dig .. xxx.example.com  but better to be safe I think and put in the dull domain.

Reply:
Make sense, thanks, Randy